clio-reporting/clio-reporting-runner/src/main/java/eu/europeana/clio/reporting/runner/config/ApplicationConfiguration.java

```diff
@@ -0,0 +1,119 @@
+package eu.europeana.clio.reporting.runner.config;
+
+import eu.europeana.clio.common.config.properties.ReportingEngineConfigurationProperties;
+import eu.europeana.clio.common.persistence.model.*;
```

Comment:
Please extend this import

Reply:
Absolutely correct, I guess I didn't properly setup my code editor formatter